package-lock.json

```diff
@@ -22302,7 +23146,7 @@
       "requires": {
         "global-agent": "^2.1.12",
         "saucelabs": "^4.6.3",
-        "webdriverio": "^6.7.0"
+        "webdriverio": "7.30.2"
```

Comment:
this is the important place where it's bumped, sauce-launcher will leverage the new 7.30.2